### DISCUSSION TITLE
Raj Music Malayalam

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#689] [Q&A]
**Channel Name:** Raj Music Malayalam
**Channel ID:** RajMusixMalayalam.in
**Country:** India
**Language:** Malayalam
**Website:** http://www.rajtvnet.in/Raj_Net/Aboutus.aspx
**Stream URL (optional):** https://www.yupptv.com/channels/raj-music-malayalam/live
**Notes (optional):** xxx

**Comment by Dum4G** [comment]
May be not 24/7
http://103.199.161.254/Content/rajtv/Live/Channel(RajTV)/Stream(01)/index.m3u8

**Comment by Niyasmkkv** [comment]
https://mhdtvworld.xyz/live/raj-music-malayalam/

**Comment by Sphinxroot** [comment]
<html>
<body>


masterManifest | https://tvstream.space/jtv/jiio.php?c=Raj_Music_Malayalam&=800&e=.m3u8

referrer | https://tvstream.space/jtv/play.php?c=Raj_Music_Malayalam


</body>
</html>

here the fun part if you grab the php file https://tvstream.space/jtv/jiio.php?c=Raj_Music_Malayalam

#EXTM3U
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:6693
#EXT-X-TARGETDURATION:6
#EXT-X-KEY:METHOD=AES-128,URI="jkey.php?key=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629188926000.key",IV=0x00000000000000000000017B533A2A30
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189482000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189490000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189496000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189502000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189508000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189514000.ts
#EXTINF:6.000,
stream.php?ts=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189520000.ts
#EXT-X-KEY:METHOD=AES-128,URI="jkey.php?key=Raj_Music_Malayalam/Raj_Music_Malayalam_800-1629189526000.key",IV=0x00000000000000000000017B534351F0


and so on


if i use this in kodi , it work both addon resolve it but not with the &=800&e=.m3u8 final

#EXTINF:-1, Raj Music Malayalam
plugin://plugin.video.sendtokodi/?https://tvstream.space/jtv/jiio.php?c=Raj_Music_Malayalam
#EXTINF:-1, Raj Music Malayalam
plugin://plugin.video.streamlink-tester/?action=play&url=https://tvstream.space/jtv/jiio.php?c=Raj_Music_Malayalam

https://i.imgur.com/1KyFoxB.png

**Comment by BellezaEmporium** [comment]
![image](https://user-images.githubusercontent.com/30985701/129757355-7fec65cd-cfe9-40f4-99cf-75deb1ea86c5.png)
My streamlink doesn't seem to like that file.

**Comment by Niyasmkkv** [comment]
Is this working? Showing 404 for me. @Dum4G @LaneSh4d0w

**Comment by Dum4G** [comment]
Jio seem to have good protection against VPNs, I was never able to get any of these working.

**Comment by BellezaEmporium** [comment]
> 
> 
> May be not 24/7
> http://103.199.161.254/Content/rajtv/Live/Channel(RajTV)/Stream(01)/index.m3u8

![image](https://user-images.githubusercontent.com/30985701/131019345-5d8f1424-84e8-4d38-b450-7e4eecf7bc04.png)

**Comment by Dum4G** [comment]
Yeah, it's like that since I posted it. The link is useless

**Comment by Niyasmkkv** [comment]
Yup. It never worked for me.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19724
